statistics/read-rtdb-gamespecs.js

```diff
@@ -0,0 +1,128 @@
+const serviceAccount = require("../../testproject-firebase-adminsdk.json");
+const admin = require("firebase-admin");
+
+admin.initializeApp({
+  credential: admin.credential.cert(serviceAccount),
+  databaseURL: "https://testproject-a6dce.firebaseio.com/",
+  storageBucket: 'testproject-a6dce.appspot.com'
+});
+
+const db = admin.database();
+const dbRef = db.ref("/gameBuilder");
+const dbTarget = db.ref("/gamePortal/gamesInfoAndSpec/gameInfo");
+const dbTargetSpec = db.ref("/gamePortal/gamesInfoAndSpec/gameSpecsForPortal");
+var dbTargetElement;
+
+
+var writeUserData = function (promisesSpecs, promisesElemes, promisesImages,itemsElems,itemsSpecs,dbTargetElem) {
```

Comment:
You don't need to pass most of the arguments here now. Remove the empty arrays that are passed and create them locally.